dynawo/sources/Solvers/SolverSIM/DYNSolverSIM.h

```diff
@@ -265,6 +265,8 @@ class SolverSIM : public Solver::Impl {
   int mxiter_;  ///< maximum number of nonlinear iterations
   int printfl_;  ///< level of verbosity of output
 
+  bool skipNextNR_;  ///< indicates if the next NR resolution could be skipped
```

Comment:
Please put the complete meaning for NR in the comment so that someone who is not familiar can understand

Reply:
Done